manifests/m/Meld/Meld/3.22.0/Meld.Meld.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=AUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Meld.Meld
+PackageVersion: 3.22.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.gnome.org/binaries/win32/meld/3.22/Meld-3.22.0-mingw.msi
+  InstallerSha256: 2789ED3B259A506D359E1C17A2764E9DACCD70FBC8365803C20612B0C9F2FBD8
+  ProductCode: '{68E6C01E-D7C1-4BED-9FDA-A00118F7CBC4}'
```

```suggestion
  ProductCode: '{F305FD7B-EE65-4D75-ADB4-A6BD7D5F86B9}'
```
This must match Windows Registry.